administration/src/project-configs/helper/getGeoDataUrlWithParams.ts

```diff
@@ -0,0 +1,12 @@
+export const GEO_SERVICE_URL = 'https://nominatim.maps.tuerantuer.org/nominatim/search'
+const getGeoDataUrlWithParams = (location: string, street: string, houseNr: string): URL => {
+  const geoServiceUrl = new URL(GEO_SERVICE_URL)
+  geoServiceUrl.searchParams.append('format', 'geojson')
+  geoServiceUrl.searchParams.append('addressdetails', '1')
```

Comment:
What is this 1 for? Can you add a comment to explain or remove if unneeded.

Reply:
ok it looks like its not needed. copied it from the backend import